nginx-transcoder/nginx.conf

```diff
@@ -105,6 +105,13 @@ http {
             return 401;
         }
 
+        #forward to custom rtmp auth url
+        location /route_to_auth {
+            allow 127.0.0.1;
+            deny all;
+            proxy_pass  ${RTMP_AUTH_URL};
+        }
```

Comment:
we should still default to /auth if $RTMP_AUTH_URL is not set

Reply:
@roddylindsay We already default to /auth if $RTMP_AUTH_URL is not set. 

In docker-compose.yml (```- RTMP_AUTH_URL=${RTMP_AUTH_URL:-http://127.0.0.1:80/auth}``` ) if no other value is selected in the .env file, we default to /auth already there.